docs/svelte-testing-library/setup.mdx

```diff
@@ -4,83 +4,89 @@ title: Setup
 sidebar_label: Setup
 ---
 
-We recommend using [Vitest](https://vitest.dev/) but you're free to use the
-library with any testing framework and runner you're comfortable with.
+We recommend using [Vitest][], but you're free to use the library with any test
+runner that's ESM compatible.
 
-## Vitest
-
-1. Install Vitest and jsdom
+[vitest]: https://vitest.dev/
 
-We're using `jsdom` here as the test environment, but you can use any other
-options e.g `happy-dom`.
+## Vitest
 
-```bash npm2yarn
-npm install --save-dev vitest jsdom
-```
+1. Add development dependencies
 
-Optionally install `@vitest/ui`, which opens a UI within a browser window to
-follow the progress and interact with your tests.
+   - [`@testing-library/svelte`][@testing-library/svelte]
+   - [`@testing-library/jest-dom`][@testing-library/jest-dom] (Optional, but
+     highly recommended)
+   - [`@sveltejs/vite-plugin-svelte`][@sveltejs/vite-plugin-svelte]
+   - [`vitest`][vitest]
+   - [`jsdom`][jsdom], [`happy-dom`][happy-dom], or other [Vitest DOM
+     environment][]
 
-```bash npm2yarn
-npm install --save-dev @vitest/ui
-```
+   ```bash npm2yarn
+   npm install --save-dev \
+     @testing-library/svelte \
+     @testing-library/jest-dom \
+     @sveltejs/vite-plugin-svelte \
+     vitest \
+     jsdom
+   ```
 
-1. Add the test scipts to your `package.json` to run the tests with Vitest
+   Optionally install [`@vitest/ui`][@vitest/ui], which opens a UI within a
+   browser window to follow the progress and interact with your tests.
 
-   ```json
-   {
-     "scripts": {
-       "test": "vitest run",
-       "test:ui": "vitest --ui",
-       "test:watch": "vitest"
-     }
-   }
+   ```bash npm2yarn
+   npm install --save-dev @vitest/ui
    ```
 
-2. To compile the Svelte components before using them in Vitest, you need to
-   install
-   [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte)
-   and Vite
+2. Add a `vitest-setup.js` file to configure cleanup and
+   [`@testing-library/jest-dom`][@testing-library/jest-dom], if using.
 
-   ```bash npm2yarn
-   npm install --save-dev @sveltejs/vite-plugin-svelte vite
+   ```ts
+   // vitest-setup.js
```

Comment:
Could you use the title's feature on this page instead of an inline file names?
https://docusaurus.io/docs/2.x/markdown-features/code-blocks#code-title

For example:
```suggestion
   ```ts title="vitest-setup.js"
```